back-end/services/UserService.js

```diff
@@ -8,4 +10,11 @@ const updateByID = async (id, body) => {
   );
 };
 
-module.exports = { findByEmail, updateByID };
+const findRoleId = async (email) => {
```

Comment:
Your service is called findROLEID but it return you a whole user object. (BAD NAMING)